packaging/nnstreamer.spec

```diff
@@ -1,7 +1,24 @@
-# Execute gbs with --define "testcoverage 1" in case that you must get unittest coverage statistics
+###########################################################################
+#
+#              Options for gbs/rpmbuild users
+#
+# gbs --define "unit_test 1"
+#       Execute all unit test cases
+#
+# gbs --define "testcoverage 1"
+# 	Generate unittest coverage statistics
+#       Use with "unit_test 1" to do it with as many cases as possible:
+#       $ gbs --define "unit_test 1" --define "testcoverage 1"
```

Comment:
```suggestion
#       $ gbs build --define "unit_test 1" --define "testcoverage 1"
```

Reply:
Applied

Reply:
I think you don't modify this line. Please check it again.

Reply:
applied. :)